kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-runtime/src/main/java/org/kie/server/services/taskassigning/runtime/command/DelegateAndSaveCommand.java

```diff
@@ -38,6 +38,9 @@
     static final String TASK_MODIFIED_ERROR_MSG = "Task: %s was modified by an external action since the last executed plan," +
             " current status is %s but the expected should be in %s";
 
+    static final String TASK_MODIFIED_ERROR_MSG_1 = "Task: %s was not found, it might have been deleted by an external action since the last executed plan." +
```

Comment:
soft suggestion: naming variables or constants should reflect their purpose. Here we use a number just to distinguish two constants from each other.